src/hooks/constants/resolvers/query/constants.tsx

```diff
@@ -0,0 +1,44 @@
+import { useCallback } from 'react';
+import { ApiPromise } from '@polkadot/api';
+import { Constants, LbpConstants, Fee } from '../../../../generated/graphql';
+import { usePolkadotJsContext } from '../../../polkadotJs/usePolkadotJs';
+import { withErrorHandler } from '../../../apollo/withErrorHandler';
+import { fetchConstants } from '../../lib/fetchConstants';
+
+export const __typenameConstants: Constants['__typename'] = 'Constants';
+export const __typenameLbpConstants: LbpConstants['__typename'] =
+  'LBPConstants';
+export const __typenameFee: Fee['__typename'] = 'Fee';
+const withTypename = (constants: Constants) => ({
```

Comment:
the constants resolver should be modular, you should not couple lbp specific constants with the overall constants resolver, please separate it

Reply:
analysis+solution:  https://github.com/galacticcouncil/Basilisk-ui/pull/109/files#r810244232